src/Raven.Client/Documents/Operations/Counters/CounterBatch.cs

```diff
@@ -82,6 +88,9 @@ public enum CounterOperationType
         GetAll
```

Comment:
let's put a note about each type here.
I wonder 
- what is `put` vs `increment` (would it override the value)
- `get` vs `getall` (if we specify a counter name)

Reply:
done